apps/console/src/features/claims/components/edit/local-claim/edit-basic-details-local-claims.tsx

```diff
@@ -618,6 +618,7 @@ export const EditBasicDetailsLocalClaims: FunctionComponent<EditBasicDetailsLoca
             <Divider hidden />
             {
                 attributeConfig.editAttributes.showDangerZone && !hideSpecialClaims
+                && claim.claimURI !== ClaimManagementConstants.EMAIL_CLAIM_URI
```

Comment:
Shall we format this as follows for better readability?
```suggestion
                attributeConfig.editAttributes.showDangerZone 
                && !hideSpecialClaims
                && claim.claimURI !== ClaimManagementConstants.EMAIL_CLAIM_URI
```

Reply:
Addressed with f287077099b17fd55f727cc5c7741fbbda4a5640.